osbs/api.py

```diff
@@ -965,3 +966,50 @@ def get_resource_quota(self, quota_name):
     @osbsapi
     def can_orchestrate(self):
         return self.build_conf.get_can_orchestrate()
+
+    @osbsapi
+    def create_config_map(self, name, data):
+        """
+        Create an ConfigMap object on the server
+
+        Raises exception on error
+
+        :param name: str, name of configMap
+        :param data: dict, dictionary of data to be stored
+        :returns: ConfigMapResponse containing the ConfigMap with name and data
+        """
+        config_data_file = os.path.join(self.os_conf.get_build_json_store(), 'config_map.json')
+        config_data = json.load(open(config_data_file))
+        config_data['metadata']['name'] = name
+        config_data['data'] = data
+
+        response = self.os.create_config_map(config_data)
+        config_map_response = ConfigMapResponse(response.json())
+        return config_map_response
+
+    @osbsapi
+    def get_config_map(self, name):
+        """
+        Get a ConfigMap object from the server
+
+        Raises exception on error
+
+        :param name: str, name of configMap to get from the server
+        :returns: ConfigMapResponse containing the ConfigMap with the requested name
+        """
+        response = self.os.get_config_map(name)
+        config_map_response = ConfigMapResponse(response.json())
+        return config_map_response
```

Comment:
Rather than a ConfigMapResponse?

The idea is to provide an interface that could grow with any changes that occur, and to have some consistency in the API: ask for a thing, get a class instance that provides access methods.